config/initializers/smtp.rb

```diff
@@ -1,13 +1,20 @@
 if Rails.env.development?
-ActionMailer::Base.delivery_method = :smtp
-ActionMailer::Base.smtp_settings = {
-  :address              => "smtp.zoho.com",
-  :port                 => 465,
-  :user_name            => 'countries_launch_academy@zoho.com',
-  :password             => 'MakeNoBugs!',
-  :authentication       => :login,
-  :ssl                  => true,
-  :tls                  => true,
-  :enable_starttls_auto => true
+  Rails.application.config.action_mailer.delivery_method = :smtp
+  Rails.application.config.action_mailer.smtp_settings = {
+  :address => "127.0.0.1",
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.